logic/src/androidMain/kotlin/com/wire/kalium/logic/util/PlatformContext.kt

```diff
@@ -16,17 +16,10 @@
  * along with this program. If not, see http://www.gnu.org/licenses/.
  */
 
-package com.wire.kalium.logic.feature.call.scenario
+package com.wire.kalium.logic.util
 
-import com.sun.jna.Pointer
-import com.wire.kalium.calling.callbacks.VideoReceiveStateHandler
-import com.wire.kalium.logic.callingLogger
+import android.content.Context
 
-class OnParticipantsVideoStateChanged : VideoReceiveStateHandler {
-    override fun onVideoReceiveStateChanged(conversationId: String, userId: String, clientId: String, state: Int, arg: Pointer?) {
-        callingLogger.i(
-            "[onVideoReceiveStateChanged] - conversationId: $conversationId | userId: $userId clientId: $clientId" +
-                    " | state: $state"
-        )
-    }
-}
+actual class PlatformContext(
+    val context: Context
+)
```

Comment:
Strange enough this one looks like it removed `OnParticipantsVideoStateChanged` and renamed/added `PlatformContext` 🤔

Reply:
I think it's just the GitHub diff that's confused, when look at the branch the `OnParticipantsVideoStateChanged` and `OnMissedCall` is still there.